ddtrace/utils/__init__.py

```diff
@@ -10,6 +13,20 @@ def get_module_name(module):
     """Returns a module's name or None if one cannot be found.
     Relevant PEP: https://www.python.org/dev/peps/pep-0451/
     """
-    if hasattr(module, '__spec__'):
+    if hasattr(module, "__spec__"):
         return module.__spec__.name
-    return getattr(module, '__name__', None)
+    return getattr(module, "__name__", None)
+
+
+# https://stackoverflow.com/a/7864317
+class classproperty(property):
```

Comment:
I don't see this function being used here. Do you intend this as a utility function we should add for other purposes?

Reply:
yeah, I think I originally thought I could pair with `@removed_property`, but just left in, didn't think much about it.

happy to remove, not a big deal.